providers/shared/components/datacatalog/id.ftl

```diff
@@ -0,0 +1,266 @@
+                                "Names" : "Enabled",
+                                "Description": "Should the column be included",
+                                "Types": BOOLEAN_TYPE,
+                                "Default": true
+                            },
+                            {
+                                "Names": "Name",
+                                "Description": "The name of the column",
+                                "Types": STRING_TYPE,
+                                "Mandatory": true
+                            },
+                            {
+                                "Names" : "Type",
+                                "Description": "The data type of the column",
+                                "Types": STRING_TYPE,
+                                "Default" : ""
+                            },
+                            {
+                                "Names": "Description",
+                                "Description": "A descrpition of the column",
+                                "Types": STRING_TYPE,
+                                "Default" : ""
+                            }
+                        ]
+                    },
+                    {
+                        "Names": "Columns",
+                        "Description" : "The base columns in the data article - name is the key of the object by default",
+                        "SubObjects": true,
+                        "Children" : [
+                            {
+                                "Names" : "Enabled",
+                                "Description": "Should the column be included",
+                                "Types": BOOLEAN_TYPE,
+                                "Default": true
+                            },
+                            {
+                                "Names": "Name",
+                                "Description": "The name of the column",
+                                "Types": STRING_TYPE,
+                                "Mandatory": true
+                            },
+                            {
+                                "Names" : "Type",
+                                "Description": "The data type of the column",
+                                "Types": STRING_TYPE,
+                                "Default" : ""
+                            },
+                            {
+                                "Names": "Description",
+                                "Description": "A descrpition of the column",
+                                "Types": STRING_TYPE,
+                                "Default" : ""
+                            }
+                        ]
+                    }
+                ]
+            },
+            {
+                "Names": "Format",
+                "Description" : "Defines the format of the data",
+                "Children": [
+                    {
+                        "Names": "SerDe",
```

Comment:
Name a bit cryptic - maybe just use "Serialisation". I know it doesn't explicitly mention deserialisation but the two are normally referred to together anyway...

Reply:
Yeah fair call, I was a bit confused at first but it does seem to be the indurstry term. However Serialisation does cover both sides I think. Updated across the PR's